scripts/configure-installer-tests-cluster.sh

```diff
@@ -10,6 +10,7 @@ KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
 KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeadmin-password"}
 # Exported to current env
 export KUBECONFIG=${KUBECONFIG:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig"}
+export TIMEOUT="20m"
```

Comment:
Love it, but can you try https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/redhat-developer_odo/1460/pull-ci-redhat-developer-odo-master-e2e-core/121